pkg/ocm/config.go

```diff
@@ -0,0 +1,157 @@
+package ocm
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/glog"
+	"github.com/mitchellh/go-homedir"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/openshift/rosa/pkg/debug"
```

Comment:
I'm not sure you were supposed to take rosa/debug and Rosa/info

Reply:
Do we need these loggers and go-homedir even? Our code should only run on POSIX systems (Mac/Linux) so we could just read `$HOME` and that's it.

Reply:
Yeah, I don't think we need it

Reply:
Ok, will remove all unnecessary stuff tomorrow.